ROADMAP.md

```diff
@@ -6,44 +6,15 @@ AsyncAPI is a specification for defining message-driven APIs. We're on a mission
 
-We're working hard to reach version 2.0.0, which will be full of great new features (and bug fixes.)
+### Release Candidates (RC)
+- Release Candidate (RC) version is published for MINOR and MAJOR version, so that users can validate, tests and give feedback for upcoming changes.
+- RC version follow naming convention as `2.1.0-rc1`, `2.1.0-rc2` ... and so on.
+- Any added functionality in RC version may change or be completely removed in the respective MINOR or MAJOR version.
+- There are no release candidates (RC) for PATCH versions.
 
-> *If you're looking for dates, check out our [Milestones](https://github.com/asyncapi/asyncapi/milestones?direction=asc&sort=due_date) page.*
+### Upcoming tasks
 
-### Finished milestones
-
-:arrow_right: [[v2.0.0] Normalize topics and support multiple schemas](https://github.com/asyncapi/asyncapi/milestone/3)
-
-Before we continue heading to version 2.0.0, let's make the biggest breaking changes in the history of AsyncAPI: topic normalization and support for multiple schemas.
-
-:arrow_right: [[v2.0.0] Improving connectivity](https://github.com/asyncapi/asyncapi/milestone/1)
-
-This milestone tackles the issues related to connecting to a server. These issues are very important to resolve for SDKs and code generators to work properly.
-
-:arrow_right: [[v2.0.0] Improve user experience](https://github.com/asyncapi/asyncapi/milestone/4) 
-
-This milestone is about improving developer/user experience for the specification and the tooling.
-
-:arrow_right: [[v2.0.0] Protocol mappings](https://github.com/asyncapi/asyncapi/milestone/2)
-
-This milestone is about figuring out how to map protocols to AsyncAPI documents.
-
-:arrow_right: [[v2.0.0] Augmentation](https://github.com/asyncapi/asyncapi/milestone/5) :white_check_mark: Current milestone
-
-This milestone is about creating mechanisms for the users to augment the AsyncAPI specification. We already have specification extensions, which provide a syntactical way to extend the spec. In this milestone, we should aim for providing the right mechanisms to extend the spec in different contexts and ways.
-
-:arrow_right: AsyncAPI version 2.0.0 Release Candidate launch
-
-There's a new AsyncAPI version and now it's time for the tooling vendors to finish implementing it.
-
-:arrow_right: :tada::tada::tada: AsyncAPI version 2.0.0 launch
-
-Official launch of AsyncAPI 2.0.0! Party hard! :beers:
-
-### Next milestones
-
-:arrow_right: [Backlog](https://github.com/asyncapi/asyncapi/projects/4)
-
-The purpose of the backlog is to temporarily group issues that may be considered for the future. Please, note that it doesn't mean they will get implemented, however, there is a big chance.
+You can check upcoming features and in-progress tasks thorugh [ZenHub Dashboard](https://app.zenhub.com/workspaces/asyncapi-initiative-5e541969da30b0bc34628b22/roadmap)
```

Comment:
Thanks for removing the junk here. Let's not point to Zenhub as you either need an account or the extension to see the information. Let's use [shapeup.asyncapi.io](https://shapeup.asyncapi.io) instead. Actually, this whole document should probably be removed and we just add links to README pointing to [shapeup.asyncapi.io](https://shapeup.asyncapi.io).

Reply:
Makes sense. 👍